### DISCUSSION TITLE
Generate UPDATE script for a particular column

### DISCUSSION THREAD

**Discussion by arunsnmimt** [dbeaver/dbeaver#34051] [Q&A]
Hello All,

I am using postgres database..However when I try to generate update statement from database editor it always creates update statement with WHERE clause as ID column..meams primary key column..Is there a way we can create update statements with different column in WHERE clause.

![Generate-SQL-from-Data-Editor.png](https://github.com/dbeaver/dbeaver/assets/43993275/a08d6ee3-66d5-4831-be02-2c62bfaeaaad)

Can I get Title field in WHERE clause

Regards,Arun

**Comment by LonwoLonwo** [comment]
Hello @arunsnmimt 

It looks like you do not have such an option.

**Comment by TB-HunBoka** [comment]
As much as I know, it works with the index column only, so if you can change the index column to be the title, then you should be able to do that.
Or create an object based on this, where the title is indexed. Then you can generate the update command by replacing the target only.
